packages/fuels/src/cli/commands/build/buildSwayProgram.ts

```diff
@@ -1,19 +1,26 @@
 import { spawn } from 'child_process';
 
 import type { FuelsConfig } from '../../types';
-import { debug, loggingConfig } from '../../utils/logger';
+import { loggingConfig, log } from '../../utils/logger';
 
 import { onForcExit, onForcError } from './forcHandlers';
 
 export const buildSwayProgram = async (config: FuelsConfig, path: string) => {
-  debug('Building Sway program', path);
+  log('Building Sway program', path);
 
   return new Promise<void>((resolve, reject) => {
     const args = ['build', '-p', path].concat(config.forcBuildFlags);
     const forc = spawn(config.forcPath, args, { stdio: 'pipe' });
     if (loggingConfig.isLoggingEnabled) {
       // eslint-disable-next-line no-console
       forc.stderr?.on('data', (chunk) => console.log(chunk.toString()));
+
+      forc.stderr?.on('data', (chunk) => {
+        if (chunk.toString().includes('Aborting')) {
+          // because forc logs out 'Aborting due to 2 compilation errors' etc.
+          onForcError(reject);
+        }
+      });
```

Comment:
I tried compiling an invalid project with `forc build` and it exited with an error code `1`, so it seems to me that we shouldn't be parsing `forc` logs but rather just throwing if `forc build` exits with a non-zero code. Line 37: `forc.on('error', onError)` should've caught this but it didn't :thinking: Did you investigate this avenue?

Reply:
`forc.on('error, onError)` doesn't do anything because `forc` sends error messages to `stdErr` alongside other normal logs

Reply:
May it be worthwhile moving this "Abort listener" to outside of the `loggingConfig.isLoggingEnabled`, after the `forc.on('error', onError);` on line 37?